doc/release_notes_0.8.rst

```diff
@@ -0,0 +1,391 @@
+.. _release_notes_0.8:
+
+ACRN v0.8 (Apr 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.8.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation.  You can either download this source code as a zip or
+tar.gz file (see the `ACRN v0.8 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.8>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.8
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.8 documents can be found at
+https://projectacrn.github.io/0.8/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+ACRN v0.8 requires Clear Linux OS version 28600 or newer.  Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.8 new features
+************************
+
+GPIO virtualization
+=========================
+
+GPIO virtualization is supported as para-virtualization based on the
+Virtual I/O Device (VIRTIO) specification. The GPIO consumers of the
+Front-end are able to set or get GPIO values, directions, and
+configuration via one virtual GPIO controller. In the Back-end, the GPIO
+command line in the launch script can be modified to map native GPIO to
+UOS.
+
+Enable QoS based on runC container
```

Comment:
Is there any good pointer we can use here for those who want to learn more about this feature?

Reply:
Current not yet. I will ask developer to provide this guide in next release.